docs: Upload Jupyter Notebook Docker Containers User Stories (#90)
thoth-tech/documentation: docs/OnTrack/Jupyter Notebook/Docker Containers User Stories.md

- As a student, I want the new features to be able to accept and submit my work quickly, so that I
  can know I have submitted successfully and do not have to do anything else for the task.

- As a student, I want the new features not to change anything in my submission, potentially
  jeopardizing it, and ensure that the staff/tutor assessing my work will see exactly what I
  intended to submit to them.

---

## Staff/Tutor

---

- As a staff member/tutor, I want my students to be able to submit Jupyter Notebook and Word
  document files easily, so that their work can be marked in a timely and easy manner, and so the
  students don’t encounter extra unnecessary pressure when submitting.

- As a staff member/tutor, I want the work converted through the use of the containers to be an
  exact replica of what the student wished to submit to me, without any erroneous changes. This
  ensures I grade exactly what the student wished me to grade when they submitted.

- As a staff member/tutor, I want to be able to view submitted Jupyter Notebook and Word document
  files in a valid PDF format document, so that I can view and mark the work.

---

## Admin/Developer

---

> - As an Admin/Developer, I want the new features to be added seamlessly to the existing OnTrack code
>   and produce no errors or runtime issues.

Reviewer: ```suggestion
- As an Admin/Developer, I want the new features to run independantly from the existing OnTrack code
  and produce no errors or runtime issues.
```

> - As an Admin/Developer, I want the containers to run smoothly and produce the intended output every
>   time, without error, so that students can use the software as intended.

Reviewer: ```suggestion
- As an Admin/Developer, I want the feature to produce the intended output every
  time, without error, so that students can use the software as intended.
```

> - As an Admin/Developer, I want the code of the containers to be easy to understand so that I know
>   what they do, how they do it, and can edit the code if need be.

Reviewer: ```suggestion
- As an Admin/Developer, I want the code and function of the the feature to be well documented and easy to understand so that I know what they do, how they do it, and can edit the code if need be.
```

- As an Admin/Developer, I want the code to be secure so that it will not provide any security
  issues for the system.

- As an Admin/Developer, I want the code to be encapsulated, so that it is easy to extract and place

>   in another area of the code if need be.

Reviewer: Not too sure about this last one for the reason of moving the code around. However, it is true that the feature should be encapsulated - though I think that is covered by the first Admin/developer story.

Author: Last Admin/Developer line omitted
